db/sql/003-function-get-impacted-accruals.sql

```diff
@@ -0,0 +1,43 @@
+-- =============================================================================
+-- Description: Returns list of Accruals impacted by TimeEntry. Covering dates
+--              from the earliest found impacted accrual - 1 day ('day before' used
+--              in calculation of totals') till the end of agreement.
+--
+-- Parameters:
+-- @tenant_id
+-- @time_entry_start_date - TimeEntry start date (it can be before the first found accrual)
+-- @time_entry_id
+-- @agreement_end_date
+--
+-- Returns: Accruals table
+-- =============================================================================
+
+CREATE OR REPLACE FUNCTION accruals.get_impacted_accruals(
+    p_tenant_id VARCHAR, time_entry_start_date DATE, time_entry_id VARCHAR, agreement_end_date DATE)
```

Comment:
I expected to see that the person id was also passed in.
What is the `p_` prefix on tenant_id

Reply:
the p_ prefix stands for parameter. We had to use this as it was getting saying tenant_id was ambiguous. 
Basically it couldn't diferentiate between the column name and the param we are sending in.